openage/convert/main.py

```diff
@@ -16,12 +16,18 @@
 from ..util.fslike.directory import CaseIgnoringDirectory
 from ..util.strings import format_progress
 
+_STANDARD_PATH_IN_32BIT_WINEPREFIX =\
+    "drive_c/Program Files/Microsoft Games/Age of Empires II/"
+_STANDARD_PATH_IN_64BIT_WINEPREFIX =\
+    "drive_c/Program Files/Microsoft Games (x86)/Age of Empires II/"
```

Comment:
nah:
`drive_c/Program Files/Microsoft Games (x86)/Age of Empires II/`
vs
`drive_c/Program Files (x86)/Microsoft Games/Age of Empires II/`